pkg/apis/admissionregistration/types.go

```diff
@@ -334,6 +339,18 @@ type Validation struct {
 	MessageExpression string
 }
 
+// Variable is the definition of a variable that is used for composition.
```

Comment:
Also update the API docs for the Validation/AuditAnnotation/MatchCondition expressions to explain that variables are accessed via the `variables.<variablename>` ?

Reply:
Added for Validation and MatchCondition. There is no doc about expression syntax with AuditAnnotation so I am skipping it.